Makefile.am

```diff
@@ -57,13 +57,41 @@ install-exec-local:
 	$(INSTALL) -d -m 750 $(DESTDIR)/$(CRM_BLACKBOX_DIR)
 	$(INSTALL) -d -m 770 $(DESTDIR)/$(CRM_LOG_DIR)
 	$(INSTALL) -d -m 770 $(DESTDIR)/$(CRM_BUNDLE_DIR)
+ # leaving a space here to work around automake's conditionals
+ ifeq ($(host_os),linux-gnu)
```

Comment:
Isn't ifeq specific to GNU make? I don't think we require it.

Reply:
Yes, we already require `gmake` on systems where `make` != `gmake`.
If only for hooking `GNUMakefile` in.

Reply:
Also explicit `./configure MAKE=gmake` unless I am not confusing that
with `libqb`.

Reply:
Do you know where we require gmake? I can't find that.

The rpm stuff is the only non-maintainer bits in our GNUMakefile, and that's optional and incidental, so it does not constitute a requirement.

Reply:
Sure we require GNU make for long 5 years already, at minimum,
because of
```
V ?= $(AM_DEFAULT_VERBOSITY)
```
https://github.com/ClusterLabs/pacemaker/blob/Pacemaker-2.0.1-rc3/Makefile.common#L4

and that's not defined in POSIX make:
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/make.html#tag_20_76_13_05

Reply:
Pacemaker targets major Linux distributions (roughly 7-10 year window) and (recent) FreeBSD. (Anything beyond that is a bonus.) Both support "?=", but BSD make doesn't support "ifeq".

Reply:
Let's not be bought into this "we don't require GNU make unless we
actually do" iterative game, since the practice shows that there's
indeed more that <make on whichever non-GNU system> doesn't
implement but we rely on.

You can check on your own that `gmake` is required with FreeBSD,
because of (for instance):
https://github.com/ClusterLabs/pacemaker/blob/Pacemaker-2.0.1-rc3/xml/Makefile.am#L39

I'll add that to INSTALL.md, since that's really less of a hassle
than getting GNU-compatible getopt to live at particular platform
(especially since `gmake` doesn't collide with `make`).

However, there's still more to do on this PR, notably, current reboot()
arrangement doesn't work on FreeBSD, since more/other headers are
required.